Vagrantfile

```diff
@@ -10,7 +10,7 @@ end
 Vagrant.configure("2") do |config|
   bind_node_modules = true?(ENV.fetch("BIND_NODE_MODULES", Vagrant::Util::Platform.windows?))
 
-  config.vm.box = "ubuntu/xenial64"
+  config.vm.box = "ubuntu/bionic64"
```

Comment:
Even if the home directories are now in the right place, do we want to use a bento box, per the recommendation: https://www.vagrantup.com/docs/boxes.html#official-boxes ?